Update prerequisites.md (#2723)
MicrosoftDocs/memdocs: memdocs/configmgr/tenant-attach/prerequisites.md

  - Tenant sync to Intune
  - Device sync to Intune
  - Device actions in the Microsoft Endpoint Manager admin center
 

> - In microsoft Multi-Geo tenants, the Geo location of the Azure tenant and the SCCM service connection point should be the same. 

Reviewer: ```suggestion
- In Microsoft multi-geo tenants, the geo location of the Azure tenant and the System Center Configuration Manager (SCCM) service connection point should be the same. 
```

> - In Microsoft multi-geo tenants, the geo location of the Azure tenant and the System Center Configuration Manager (SCCM) service connection point should be the same. 

Reviewer: ```suggestion
- In Microsoft Multi-Geo tenants, the geo location of the Azure tenant and the System Center Configuration Manager (SCCM) service connection point should be the same. 
```

> - In Microsoft Multi-Geo tenants, the geo location of the Azure tenant and the System Center Configuration Manager (SCCM) service connection point should be the same. 

Reviewer 2: Don't use "System Center Configuration Manager" unless specifically referring to the previous product version. In this instance, there's no need to refer to the branding, since the article is already in product context (Microsoft Endpoint Manager and specifically Microsoft Endpoint Configuration Manager). So just remove "...the System Center Confiuguration Manager (SCCM)"

```suggestion
- The geographic location of the Azure tenant and the service connection point should be the same. 
```
